src/compass/utils/h5_helpers.py

```diff
@@ -398,6 +398,20 @@ def metadata_to_h5group(parent_group, burst, cfg):
     # create metadata group to write datasets to
     meta_group = parent_group.require_group('metadata')
 
+    # RFI information - placeholder
+    rfi_items = [
+        Meta('is_rfi_info_available', True,
+             'Whether RFI information is available'),
+        Meta('rfi_mitigation_performed', '',
+             'Whether or not the RFI mitigation step was performed'),
+        Meta('rfi_mitigation_domain', '',
+             'in what domain the RFI mitigation step was performed'),
+        Meta('rfi_burst_report', [],
```

Comment:
What type of information is the `rfi_burst_report` suppose to have? We do process one burst and from the name of the metadata field it seems that there is an info connected to multiple bursts

Reply:
`rfi_burst_report` would be a list of `dict`, based on the current RFI reader implementation in `s1-reader`.
The reader finds what report corresponds to what burst based on `azimuthTime`, which needs to be reviewed if that is a correct way of doing it.